services/QuillLMS/app/services/demo/create_admin_report.rb

```diff
@@ -70,7 +73,7 @@ def reset
       teacher = find_or_create_teacher_data(row['Teacher'], school, subscription)
 
       # create classroom data
-      classroom = Classroom.create(name: row['Classroom'])
+      classroom = Classroom.create(name: row['Classroom'], grade: (GRADE_MIN..GRADE_MAX).to_a.sample.to_s)
       all_classrooms.push(classroom)
       ClassroomsTeacher.create(classroom: classroom, user: teacher, role: ClassroomsTeacher::ROLE_TYPES[:owner])
```

Comment:
You can extract a constant here: `OWNER_ROLE = ClassroomsTeacher::ROLE_TYPES[:owner]` that way the loop here just refers to CONSTANT once.  This also makes it clear at the top of the file a dependency on this file.